app/models/api_key.rb

```diff
@@ -4,6 +4,8 @@ class ApiKey < ActiveRecord::Base
   class KeyAccessError < StandardError
   end
 
+  attr_accessor :scope_mode
```

Comment:
In a separate task, I'm planning to start storing this so we can have a better view of existing API keys, but for now it's transient.